### ISSUE TITLE
Wrong deb package category

### ISSUE THREAD

**Issue by a2nt** [Foundry376/Mailspring#678]
MailSpring goes into development category while it shall be inside mail category
![screenshot from 2018-02-10 15-33-45](https://user-images.githubusercontent.com/672794/36060240-0e7a7c62-0e78-11e8-9466-87e7990257f7.png)

**Comment by bengotow** [comment]
Hey! Thanks for reporting this—do you know if this will be fixed in the next release by https://github.com/Foundry376/Mailspring/commit/44f8eb7b3f1ff66c2802f41fe3177b1a061d482e#diff-aa8175aa9f60f17e92533be1c3545826? I'm not sure if there's anything else we need to do to correct this.

**Comment by a2nt** [comment]
Hey @bengotow 
Nope it won't be fixed, the problem is here: https://github.com/Foundry376/Mailspring/blob/master/app/build/resources/linux/debian/control.in
Package section defined as devel, but it looks like a template and I don't know where variable comes from.

**Comment by bengotow** [comment]
Ahh thanks for point that out—I'll check that out and see if I can get it fixed. Believe it or not, some of this stuff is still a holdover from when Mailspring was based on Atom!

**Comment by bengotow** [comment]
Looks like the list of available values is here http://www.sosst.sk/doc/debian-policy/policy.html/ch-archive.html#s-subsections:

```
 The Debian archive maintainers provide the authoritative list of subsections. At present, they are: admin, base, comm, contrib, devel, doc, editors, electronics, embedded, games, gnome, graphics, hamradio, interpreters, kde, libs, libdevel, mail, math, misc, net, news, non-US, non-free, oldlibs, otherosfs, perl, python, science, shells, sound, tex, text, utils, web, x11. 
```

Seems like mailspring definitely goes under `mail` and not any others (hamradio?!) I think the template is populated here: https://github.com/Foundry376/Mailspring/blob/master/app/build/tasks/installer-linux-task.js#L97 so we should be able to swap it out!

**Comment by bengotow** [comment]
Hey! The fix for this shipped in version 1.1.4 this morning - cheers!
